src/topology/algebra/multilinear.lean

```diff
@@ -34,10 +34,9 @@ especially when defining iterated derivatives.
 open function fin set
 open_locale big_operators
 
-universes u v w w₁ w₂ w₃ w₄
-variables {R : Type u} {ι : Type v} {n : ℕ}
-{M : fin n.succ → Type w} {M₁ : ι → Type w₁} {M₂ : Type w₂} {M₃ : Type w₃} {M₄ : Type w₄}
-[decidable_eq ι]
+universes u v w w₁ w₁' w₂ w₃ w₄
+variables {R : Type u} {ι : Type v} {n : ℕ} {M : fin n.succ → Type w} {M₁ : ι → Type w₁}
+  {M₁' : ι → Type w₁'} {M₂ : Type w₂} {M₃ : Type w₃} {M₄ : Type w₄} [decidable_eq ι]
```

Comment:
Perhaps out of scope, but I think it would be clearer if the `ι → Type` and `Type` variables had different naming conventions

Reply:
I agree but I don't want to mix too many style changes into a feature PR.